.pre-commit-config.yaml

```diff
@@ -91,3 +91,33 @@ repos:
     - id: add-license-headers
       args:
       - --start_year=2021
+
+- repo: local
+  hooks:
+    - id: check-vulnerabilities-dependencies
+      name: Check Vulnerabilities Dependencies
+      entry: python -c "import os; import subprocess; subprocess.run(['python', 'requirements/requirements_vulnernabilities.py'])"
+      language: python
+      types: [python]
+      pass_filenames: false
+      always_run: true
+
+    - id: check-vulnerabilities
+      name: Check Vulnerabilities
+      entry: python -c 
+        "import os; 
+         import subprocess;
+         result = []; 
+         result.append(subprocess.run(['python', 'check_vulnerabilities.py', '--run-local'])) 
+         if os.environ.get('DEPENDENCY_CHECK_TOKEN') 
+         else print('DEPENDENCY_CHECK_TOKEN is not set');
```

Comment:
@hpohekar We should skip the check without failure if the user hasn't set it up but also consider not having it in pre-commit. I favour the latter because it all seems heavyweight and risky for minimal payoff.

Apologies, I did not see your prior question here.

cc: @mkundu1 @RobPasMue

Reply:
@seanpearsonuk Removed.